linux/install.sh

```diff
@@ -49,11 +49,11 @@ su - omero -c "bash -eux setup_$OMEROVER.sh"
 if [ $DISTRO = centos6 ]; then
```

Comment:
Shell safety note: never use unquoted variable expansion in any place you don't want it to be split into separate tokens